draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -395,6 +395,12 @@ map2curve(alpha)
 where alpha is a message to encode on a curve. 
 
 ## Encoding Variants
+| Curve | Possible Encodings |
```

Comment:
Some explanatory text or prologue before this table would be nice. Also, we don't have Elligator Squared or DE defined, so perhaps either add sections for them or remove them? And what is TI?